app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
```

Comment:
Undo and redo typing, just like in a browser. They're not going to be heavily used, but I feel like people expect that to be in any app with an Edit menu.